### PR TITLE
Update get-geolocation-data-to-find-a-users-gps-coordinates.english.md

### PR DESCRIPTION
**Pull request by Vrajsingh**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-geolocation-data-to-find-a-users-gps-coordinates.english.md [hunk 1]
```diff
@@ -71,10 +71,23 @@ tests:
 </section>
 
 ## Solution
-<section id='solution'>
```
Comment (RandellDawson) [inline curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-geolocation-data-to-find-a-users-gps-coordinates.english.md]: Please add the following back in:
```suggestion
<section id='solution'>
```

File: curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-geolocation-data-to-find-a-users-gps-coordinates.english.md [hunk 2]
```diff
@@ -71,10 +71,23 @@ tests:
...
+
 
 ```js
```
Comment (RandellDawson) [inline curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-geolocation-data-to-find-a-users-gps-coordinates.english.md]: Please change the following to html:
````suggestion
```html
````

File: curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-geolocation-data-to-find-a-users-gps-coordinates.english.md [hunk 3]
```diff
@@ -71,10 +71,23 @@ tests:
...
+}
+  
+  // Add your code above this line
+</script>
+<h4>You are here:</h4>
+<div id="data">
+
+</div>
 
-</section>
```
Comment (RandellDawson) [inline curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-geolocation-data-to-find-a-users-gps-coordinates.english.md]: Please add the following back in:
```suggestion
</section>
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@Vrajsingh Please commit the following changes:

In the future, make sure you setup the repo locally and run the following before creating a PR.  If all the test pass, then you can create a PR.
```bash
npm run test:curriculum
```

**Comment by raisedadead** [comment]
Thanks for the PR @Vrajsingh, closing this as stale. Please open a fresh PR if you are still interested.

**Pull request closed without merging**